tests/py/dynamo/conversion/test_index_aten.py

```diff
@@ -2,11 +2,10 @@
 
 import torch
 import torch.nn as nn
+from harness import DispatchTestCase
```

Comment:
Requires switch back to `.harness` to pass CI

Reply:
Ok corrected, wil wait for CI then will merge.